xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
```

Comment:
I don't like all caps for readability and accessibility. 

I also prefer the original title. This part is about the xAPI, its not about learning experiences.